build_tools/benchmarks/convperf/build_and_run_convperf.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Sets up `convperf` (https://github.com/nod-ai/convperf).
+#
+# `convperf` benchmarks convolution workloads on IREE and other backends such
+# as libxsmm. IREE is included as a submodule.
+#
+# Currently x86 CPU is supported.
+#
+# Usage:
+#    ./build_and_run_convperf.sh \
+#        <convperf build dir> \
+#        <convperf results dir> \
+#        <iree SHA> \
+#        <convperf repo dir> (optional)
+
+set -xeuo pipefail
+
+export BUILD_DIR=$1
+export RESULTS_DIR=$2
+export IREE_SHA=${3:-"origin/main"}
+export REPO_DIR=${4:-${CONVPERF_REPO_DIR}}
+
+pushd ${REPO_DIR}
```

Comment:
Throughout the file, please quote variable expansions: https://google.github.io/styleguide/shellguide.html#s5.7-quoting

Reply:
Added quotes according to the guide